app/views/pickup_checklists/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="content-wrapper-large content-container">
+  <h3>
+    <%= t(".header") %>
+    <%= link_to(
+      zone_scheduled_pickup_path(@pickup_checklist.zone, @pickup_checklist),
+      class: "pickup-checklist-date",
+    ) do %>
+      <%= l(@pickup_checklist.date) %>
+    <% end %>
```

Comment:
@geoffharcourt are you suggesting a `/pickups/:id` path whose controller handles the redirection?